kernel/kernel-api/src/main/java/io/delta/kernel/internal/SnapshotImpl.java

```diff
@@ -64,6 +65,7 @@ public class SnapshotImpl implements Snapshot {
   private Lazy<Optional<List<Column>>> lazyClusteringColumns;
 
   // TODO: Do not take in LogReplay as a constructor argument.
+  // TODO: Also take in clustering columns for post-commit snapshot
```

Comment:
Currently we lazily load these from DM, so we will need to put that here most likely